registration/registration-services/src/main/java/io/mosip/registration/util/acktemplate/TemplateGenerator.java

```diff
@@ -575,4 +614,169 @@ private String getImage(String imagePath) {
+		List<SyncJobDef> syncJobs = masterSyncServiceImpl.getSyncJobs();
+		for (SyncJobDef syncJob : syncJobs) {
+			SyncControl syncControl = jobConfigurationService.getSyncControlOfJob(syncJob.getId());
+			if (syncControl != null && syncControl.getLastSyncDtimes() != null) {
+				Map<String, Object> job = new LinkedHashMap<>();
+				job.put(RegistrationConstants.DASHBOARD_ACTIVITY_NAME, trimJobName(syncJob.getName()));
+				job.put(RegistrationConstants.DASHBOARD_ACTIVITY_VALUE, getLocalZoneTime(syncControl.getLastSyncDtimes().toString()));
+				activities = addToJobList(activities, syncJob.getJobType(), job);
+			}
+		}
+		Map<String, Object> clientVersion = new LinkedHashMap<>();
+		clientVersion.put(RegistrationConstants.DASHBOARD_ACTIVITY_NAME, RegistrationConstants.DASHBOARD_REG_CLIENT);
+		clientVersion.put(RegistrationConstants.DASHBOARD_ACTIVITY_VALUE, softwareUpdateHandler.getCurrentVersion());
+		Map<String, Object> schemaVersion = new LinkedHashMap<>();
+		schemaVersion.put(RegistrationConstants.DASHBOARD_ACTIVITY_NAME, RegistrationConstants.DASHBOARD_ID_SCHEMA);
+		schemaVersion.put(RegistrationConstants.DASHBOARD_ACTIVITY_VALUE, String.valueOf(identitySchemaServiceImpl.getLatestEffectiveSchemaVersion()));
+		List<Map<String, Object>> versionsList = new ArrayList<>();
+		versionsList.add(clientVersion);
+		versionsList.add(schemaVersion);
+		activities.put(RegistrationConstants.DASHBOARD_VERSION, versionsList);
+		Map<String, Object> lastSWUpdate = new LinkedHashMap<>();
+		lastSWUpdate.put(RegistrationConstants.DASHBOARD_ACTIVITY_NAME, RegistrationConstants.DASHBOARD_LAST_SW_UPDATE);
+		if (ApplicationContext.map().containsKey(RegistrationConstants.LAST_SOFTWARE_UPDATE)) {
+			lastSWUpdate.put(RegistrationConstants.DASHBOARD_ACTIVITY_VALUE, getLocalZoneTime(ApplicationContext
+					.getStringValueFromApplicationMap(RegistrationConstants.LAST_SOFTWARE_UPDATE)));
+		} else {
+			lastSWUpdate.put(RegistrationConstants.DASHBOARD_ACTIVITY_VALUE, getLocalZoneTime(applicationStartTime));
+		}
+		Map<String, Object> appInstalledTime = new LinkedHashMap<>();
+		appInstalledTime.put(RegistrationConstants.DASHBOARD_ACTIVITY_NAME, RegistrationConstants.APP_INSTALLED_TIME);
+		if (ApplicationContext.map().containsKey(RegistrationConstants.REGCLIENT_INSTALLED_TIME)) {
+			appInstalledTime.put(RegistrationConstants.DASHBOARD_ACTIVITY_VALUE, getLocalZoneTime(ApplicationContext
+					.getStringValueFromApplicationMap(RegistrationConstants.REGCLIENT_INSTALLED_TIME)));
+		} else {
+			appInstalledTime.put(RegistrationConstants.DASHBOARD_ACTIVITY_VALUE, "-");
+		}
+		List<Map<String, Object>> updateList = new ArrayList<>();
+		updateList.add(appInstalledTime);
+		updateList.add(lastSWUpdate);
+		activities.put(RegistrationConstants.DASHBOARD_UPDATES, updateList);
+		return activities;
+	}
+	
+	private String trimJobName(String jobName) {		
+		return jobName.substring(0, jobName.lastIndexOf(" "));
```

Comment:
why are we not calling trim() directly ?

Reply:
Added this method just to show the purpose clearly. Removed and called trim() directly as it was called only once.